src/Dependency/TagDependency.php

```diff
@@ -47,7 +47,7 @@ final class TagDependency extends Dependency
      * For a single tag, you may specify it as a string.
      * @param int|null $ttl The TTL value of this item. null means infinity.
      */
-    public function __construct(array|string $tags, int $ttl = null)
+    public function __construct(array|string $tags, int|null $ttl = null)
```

Comment:
Maybe `public function __construct(array|string $tags, ?int $ttl = null)`?

Reply:
I kept it with the same style of the package code, which uses type union, which is explicit.